src/joypad.c

```diff
@@ -385,7 +386,20 @@ static void joypad_identify_callback(uint64_t *out_dwords, void *ctx)
                 device->rumble_method = JOYPAD_RUMBLE_METHOD_NONE;
                 device->rumble_active = false;
             }
-            if (accessory_changed)
+            if (sys_bbplayer())
+            {
+                // BBPlayer only supports Controller Paks.
+                // Controller Paks must be set up in the iQue menu.
+                if (bb_save_size( port ) == 0)
```

Comment:
This relies on bb_save_t defining the values for controller pads *exactly* as the first 4. It is an implicit assumption that's easy to break in a refactoring, and it affects a very rarely tested codepath.

```suggestion
                if (bb_save_size( BB_SAVE_PAK1 + port ) == 0)
```

Reply:
Fixed in https://github.com/DragonMinded/libdragon/pull/680/commits/ecd5a52bedf72b5d45c5b10402ddc09d78f760b6